install_ubuntu/tutorial_11.0.md

```diff
@@ -61,35 +61,6 @@ installation](http://gazebosim.org/tutorials?tut=ros_wrapper_versions&cat=connec
         # For developers that work on top of Gazebo, one extra package
         sudo apt-get install libgazebo11-dev
 
-    If you see the error below:
-
-        $ sudo apt-get install gazebo11
-        Reading package lists... Done
-        Building dependency tree
-        Reading state information... Done
-        E: Unable to locate package gazebo11
-
-    It's possible the version of Gazebo you are looking for is not supported on the version of OS you are using.
-    For example, installing gazebo11 on Ubuntu Xenial (16.04) will produce the error above.
-    Hint: Take a look at "Project Status" section at [http://gazebosim.org/#status](http://gazebosim.org/#status),
-    next to each version is the supported ubuntu versions and ROS versions.
-
-
 1. Check your installation
 
         gazebo
-
-## Gazebo in different deb packages
```

Comment:
It's not clear to me what's wrong with this section

Reply:
Nothing was bad in the section, it was created when we first split up the gazebo monolithic package to help people understanding what's going on. That happened years ago, I found no reason to keep the section here and found that people can be confused with it. Just a personal opinion.